src/c/reqdata.c

```diff
@@ -103,8 +103,17 @@ extern const char *edgex_reqdata_get (const edgex_reqdata_t *data, const char *n
   if (data->json)
   {
     JSON_Value *value = json_object_get_value(data->json, name);
-    if (value) {
-      result = json_serialize_to_string(value);
+    if (value) 
+    {
+      if(json_value_get_type(value) == JSONString)
+      {
+        //The slash of a string cannot be removed using json_serialize_to_string
```

Comment:
IMO, this comment is superfluous and can be removed. The code itself clearly explains its purpose.